probePlugin/src/main/scala/org/virtuslab/ProbePluginExtensions.scala

```diff
@@ -0,0 +1,34 @@
+package org.virtuslab
+
+import java.nio.file.Path
+
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.roots.ModuleRootManager
+import org.jetbrains.jps.model.java.JavaResourceRootType.{RESOURCE, TEST_RESOURCE}
+import org.jetbrains.jps.model.java.JavaSourceRootType.{SOURCE, TEST_SOURCE}
+import org.virtuslab.ideprobe.protocol.{ContentRoot, ModuleRef, ProjectRef}
+import org.virtuslab.ideprobe.Extensions._
+
+object ProbePluginExtensions {
+  final implicit class ModuleExtension(module: Module) {
+    def toRef: ModuleRef = {
+      val project = module.getProject
+      ModuleRef(module.getName, ProjectRef(project.getName))
+    }
+
+    def dependencies: Array[Module] = roots.getDependencies
+
+    def contentRoots(kind: ContentRoot): (ContentRoot, Set[Path]) = {
```

Comment:
this is quite odd api.
```scala
def returnX(a: A): (A, X) = {
  a -> getX
}
```
It feels like creating tuple with param belongs outside of it

Reply:
It no longer creates a tuple here, but rather on the call site